docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,192 @@
+---
+comments: true
+---
+
+# Deploying Applications with Flux and Keptn
+
+[Flux](https://fluxcd.io/)
+is a tool that keeps Kubernetes clusters synchronized with sources
+of configuration (such as Git repositories) and automates updates to
+the configuration when new code is deployed.
+
+This section shows how to add Keptn
+[pre- and post-deployment tasks](../guides/tasks.md)
+to an existing Flux use case that runs
+[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/).
+Adding Keptn makes it simpler and more straight-forward
+to run the Flux pre and post-deployment jobs
+and provides added observability out of the box.
+
+## High-level structure of the Git repository
+
+Flux uses a GitOps approach for continuous delivery.
+The Git
+repository structure for our use case looks like the following:
+
+```markdown
+├── apps
+│   └── dev
+│       ├── podtato-head.yaml
+│       └── kustomize.yaml
+└── clusters
+    └── dev
+        ├── flux-system
+        │   ├── gotk-components.yaml
+        │   ├── gotk-sync.yaml
+        │   └── kustomize.yaml
+        ├── podtato-head-source.yaml
+        └── podtato-head-kustomization.yaml
+```
+
+The `apps` directory contains application manifests, that will be deployed.
+The `clusters` directory contains Flux configuration manifests and custom
+resources, that are needed for the delivery.
+The `apps` and `clusters` directories can live in two separate repositories,
+but for simplicity of this excercise, we will keep them in a single one.
+
```

Comment:
```suggestion

You see that the Keptn application runs any pre/post-deployment tasks
rather than using the Flux `pre-deploy` and `post-deploy` directories in the Git repository.
The Keptn process is easier to implement
and contains more information than the Flux jobs do.

```